hypervisor/arch/x86/guest/instr_emul.c

```diff
@@ -233,7 +233,7 @@ static enum cpu_reg_name gpr_map[16] = {
 	CPU_REG_R15
 };
 
-static uint64_t size2mask[] = {
+static uint64_t size2mask[9] = {
```

Comment:
Did u see the difference with gcc？

Reply:
Without explicitly declaring the array size of 'size2mask[]', MISRAC will treat the array size as 4.